# unattended-upgrades po translation to Spanish
# Copyright (C) 2012 Software in the Public Interest
# This file is distributed under the same license as the unattended-upgrades package.
#
# Changes:
#   - Initial translation
#       Omar Campagne <ocampagne@gmail.com>, 2012
#
#   - Updates
#       TRANSLATOR
#
# Traductores, si no conocen el formato PO, merece la pena leer la
# documentación de gettext, especialmente las secciones dedicadas a este
# formato, por ejemplo ejecutando:
#       info -n '(gettext)PO Files'
#       info -n '(gettext)Header Entry'
#
# Equipo de traducción al español, por favor lean antes de traducir
# los siguientes documentos:
#
#   - El proyecto de traducción de Debian al español
#     http://www.debian.org/intl/spanish/
#     especialmente las notas y normas de traducción en
#     http://www.debian.org/intl/spanish/notas
#
#   - La guía de traducción de po's de debconf:
#     /usr/share/doc/po-debconf/README-trans
#     o http://www.debian.org/intl/l10n/po-debconf/README-trans
#
msgid ""
msgstr ""
"Project-Id-Version: unattended-upgrades 0.79\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-07-02 12:27+0200\n"
"PO-Revision-Date: 2012-07-12 14:04+0200\n"
"Last-Translator: Omar Campagne <ocampagne@gmail.com>\n"
"Language-Team: Debian l10n Spanish <debian-l10n-spanish@lists.debian.org>\n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Virtaal 0.7.1\n"

#: ../unattended-upgrade:162
#, python-format
msgid "Progress: %s %% (%s)"
msgstr "Progreso: %s %% (%s)"

#: ../unattended-upgrade:418 ../unattended-upgrade:501
#: ../unattended-upgrade-shutdown:161
msgid "All upgrades installed"
msgstr "Se han instalado todas las actualizaciones"

#: ../unattended-upgrade:420 ../unattended-upgrade:493
msgid "Installing the upgrades failed!"
msgstr "Fallo al instalar actualizaciones"

#: ../unattended-upgrade:421 ../unattended-upgrade:494
#, python-format
msgid "error message: '%s'"
msgstr "Mensaje de fallo: «%s»"

#: ../unattended-upgrade:422 ../unattended-upgrade:495
#, python-format
msgid "dpkg returned a error! See '%s' for details"
msgstr "Se ha detectado un fallo de dpkg. Para más detalles, consulte «%s»"

#: ../unattended-upgrade:795
#, fuzzy
#| msgid ""
#| "No '/usr/bin/mail', can not send mail. You probably want to install the "
#| "'mailx' package."
msgid ""
"No '/usr/bin/mail' or '/usr/sbin/sendmail',can not send mail. You probably "
"want to install the 'mailx' package."
msgstr ""
"No se ha detectado «/usr/bin/mail», no se puede enviar correo. Considere "
"instalar el paquete «mailx»."

#: ../unattended-upgrade:811
#, fuzzy, python-format
#| msgid "[reboot required] unattended-upgrades result for '%s'"
msgid "[reboot required] unattended-upgrades result for '%s': %s"
msgstr "[Se requiere un reinicio] Resultado de unattended-upgrades para «%s»"

#: ../unattended-upgrade:814
#, fuzzy, python-format
#| msgid "unattended-upgrades result for '%s'"
msgid "unattended-upgrades result for '%s': '%s'"
msgstr "Resultado de unattended-upgrades para «%s»"

#: ../unattended-upgrade:816
#, python-format
msgid ""
"Unattended upgrade returned: %s\n"
"\n"
msgstr ""
"Valor devuelto por unattended-upgrade: %s\n"
"\n"

#: ../unattended-upgrade:819
msgid ""
"Warning: A reboot is required to complete this upgrade.\n"
"\n"
msgstr ""
"Advertencia: Se requiere reinciar para completar la actualización.\n"
"\n"

#: ../unattended-upgrade:821
#, fuzzy
#| msgid "Packages that are upgraded:\n"
msgid "Packages that were upgraded:\n"
msgstr "Paquetes que actualizar:\n"

#: ../unattended-upgrade:823
#, fuzzy
#| msgid "Packages that are upgraded:\n"
msgid "Packages that attempted to upgrade:\n"
msgstr "Paquetes que actualizar:\n"

#: ../unattended-upgrade:827
msgid "Packages with upgradable origin but kept back:\n"
msgstr ""
"Paquetes con una actualización disponible en la fuente pero que no se "
"actualizarán:\n"

#: ../unattended-upgrade:832
msgid "Package installation log:"
msgstr "Registro de instalación de paquete:"

#: ../unattended-upgrade:835
msgid "Unattended-upgrades log:\n"
msgstr "Registro de unattended-upgrades:\n"

#: ../unattended-upgrade:855
#, python-format
msgid "Writing dpkg log to '%s'"
msgstr "Escribiendo el registro de dpkg en «%s»"

#: ../unattended-upgrade:987
#, python-format
msgid "Found %s, but not rebooting because %s is logged in."
msgid_plural "Found %s, but not rebooting because %s are logged in."
msgstr[0] ""
msgstr[1] ""

#: ../unattended-upgrade:1033
#, python-format
msgid "package '%s' upgradable but fails to be marked for upgrade (%s)"
msgstr ""
"El paquete «%s» es actualizable, pero no se puede marcar para la "
"actualización (%s)"

#: ../unattended-upgrade:1097
#, python-format
msgid "Packages that are auto removed: '%s'"
msgstr "Paquetes eliminados de forma automática: «%s»"

#: ../unattended-upgrade:1103
#, fuzzy
#| msgid "Packages that are auto removed: '%s'"
msgid "Packages were successfully auto-removed"
msgstr "Paquetes eliminados de forma automática: «%s»"

#: ../unattended-upgrade:1105
#, fuzzy
#| msgid "Installing the upgrades failed!"
msgid "Auto-removing the packages failed!"
msgstr "Fallo al instalar actualizaciones"

#: ../unattended-upgrade:1106
#, fuzzy, python-format
#| msgid "error message: '%s'"
msgid "Error message: '%s'"
msgstr "Mensaje de fallo: «%s»"

#: ../unattended-upgrade:1107
#, fuzzy, python-format
#| msgid "dpkg returned a error! See '%s' for details"
msgid "dpkg returned an error! See '%s' for details"
msgstr "Se ha detectado un fallo de dpkg. Para más detalles, consulte «%s»"

#: ../unattended-upgrade:1125
#, python-format
msgid "Initial blacklisted packages: %s"
msgstr "Paquetes inicialmente bloqueados: %s"

#: ../unattended-upgrade:1130
#, fuzzy, python-format
#| msgid "Initial blacklisted packages: %s"
msgid "Initial whitelisted packages: %s"
msgstr "Paquetes inicialmente bloqueados: %s"

#: ../unattended-upgrade:1133
msgid "Starting unattended upgrades script"
msgstr "Iniciando script de unattended-upgrades"

#: ../unattended-upgrade:1136
#, python-format
msgid "Allowed origins are: %s"
msgstr "Fuentes permitidas: %s"

#: ../unattended-upgrade:1152 ../unattended-upgrade:1153
msgid "Unclean dpkg state detected, trying to correct"
msgstr "Se ha detectado un estado no limpio de dpkg, intentando corregir"

#: ../unattended-upgrade:1164
#, python-format
msgid ""
"dpkg --configure -a output:\n"
"%s"
msgstr ""
"dpkg --configure -a output:\n"
"%s"

#: ../unattended-upgrade:1173
msgid "Lock could not be acquired (another package manager running?)"
msgstr ""
"No se puede establecer el bloqueo (¿otro gestor de paquetes ejecutándose?)"

#: ../unattended-upgrade:1175
msgid "Cache lock can not be acquired, exiting"
msgstr "No se puede establecer el bloqueo de la caché, saliendo"

#: ../unattended-upgrade:1182 ../unattended-upgrade:1183
#: ../unattended-upgrade:1363 ../unattended-upgrade:1364
msgid "Cache has broken packages, exiting"
msgstr "La caché tiene paquetes rotos, saliendo"

#: ../unattended-upgrade:1226
#, python-format
msgid "GetArchives() failed: '%s'"
msgstr "Fallo de GetArchives(): «%s»"

#: ../unattended-upgrade:1236 ../unattended-upgrade:1237
#, fuzzy, python-format
#| msgid "An error ocured: '%s'"
msgid "An error occurred: '%s'"
msgstr "Se ha detectado un fallo: «%s»"

#: ../unattended-upgrade:1239 ../unattended-upgrade:1241
#, python-format
msgid "The URI '%s' failed to download, aborting"
msgstr "Fallo al descargar desde el URI «%s», interrumpiendo"

#: ../unattended-upgrade:1245
#, python-format
msgid "Download finished, but file '%s' not there?!?"
msgstr "La descarga ha finalizado, pero no se encuentra el fichero «%s»"

#: ../unattended-upgrade:1261 ../unattended-upgrade:1265
#, python-format
msgid "Package '%s' has conffile prompt and needs to be upgraded manually"
msgstr ""
"El paquete «%s» solicita confirmación a través de un conffile, y se debe "
"actualizar de forma manual"

#: ../unattended-upgrade:1293
#, python-format
msgid "package '%s' not upgraded"
msgstr "No se ha actualizado el paquete «%s»"

#: ../unattended-upgrade:1311
#, fuzzy
#| msgid "No packages found that can be upgraded unattended"
msgid ""
"No packages found that can be upgraded unattended and no pending auto-"
"removals"
msgstr "No se han encontrado paquetes actualizables sin supervisión"

#: ../unattended-upgrade:1337
#, fuzzy, python-format
#| msgid "Packages that are upgraded: %s"
msgid "Packages that will be upgraded: %s"
msgstr "Paquetes que se han actualizado: %s"

#: ../unattended-upgrade:1406
msgid "print debug messages"
msgstr "Mostrar mensajes de depuración de fallos"

#: ../unattended-upgrade:1409
#, fuzzy
#| msgid "print debug messages"
msgid "make apt/libapt print verbose debug messages"
msgstr "Mostrar mensajes de depuración de fallos"

#: ../unattended-upgrade:1412
#, fuzzy
#| msgid "print debug messages"
msgid "print info messages"
msgstr "Mostrar mensajes de depuración de fallos"

#: ../unattended-upgrade:1415
msgid "Simulation, download but do not install"
msgstr "Simulacro, descarga pero no instala"

#: ../unattended-upgrade:1418
msgid "Upgrade in minimal steps (and allow interrupting with SIGINT"
msgstr "Actualizar en pequeños pasos (y permitir la interrupción con SIGINT)"

#: ../unattended-upgrade:1427
msgid "You need to be root to run this application"
msgstr ""
"Necesita tener privilegios de administrador para ejecutar esta aplicación"

#: ../unattended-upgrade-shutdown:72
msgid "Unattended-upgrade in progress during shutdown, sleeping for 5s"
msgstr "Unattended-upgrade en proceso durante el apagado, esperando 5s"

#: ../unattended-upgrade-shutdown:125
#, fuzzy
#| msgid "unattended-upgrades result for '%s'"
msgid "Running unattended-upgrades in shutdown mode"
msgstr "Resultado de unattended-upgrades para «%s»"

#: ../unattended-upgrade-shutdown:153
#, python-format
msgid "Giving up on lockfile after %s delay"
msgstr "Finalizando el bloqueo del fichero después de un retraso de %s"

#~ msgid "Progress: %s %%: (%s)"
#~ msgstr "Progreso: %s %%: (%s)"
